### PR TITLE
All the new columns to be added into telemetry for Beta 2

### PR DESCRIPTION
**Pull request by ncloudioj**

### CODE REVIEW

File: migrations/versions/44fe26926437_.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    sa.Column('date', sa.Date(), nullable=False),
+    sa.Column('locale', sa.String(length=14), nullable=False),
+    sa.Column('country_code', sa.String(length=5), nullable=False),
+    sa.Column('os', sa.String(length=64), nullable=False),
+    sa.Column('browser', sa.String(length=64), nullable=False),
+    sa.Column('version', sa.String(length=64), nullable=False),
+    sa.Column('device', sa.String(length=64), nullable=False)
+    )
+    op.add_column(u'activity_stream_stats_daily', sa.Column('page', sa.String(length=16), nullable=False))
+    op.add_column('activity_stream_stats_daily', sa.Column('date', sa.Date(), nullable=False))
```
Comment (tspurway) [inline migrations/versions/44fe26926437_.py:62]: nit: is there any way to populate this column with `trunc(receive_at)` ?

**Comment by tspurway** [comment]
r+ with nit

File: ddl/migrations/redshift/1458766468.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
+BEGIN;
+
+ALTER TABLE activity_stream_stats_daily ADD COLUMN date DATE;
+UPDATE activity_stream_stats_daily SET date = TRUNC(receive_at);
```
Comment (ncloudioj) [inline ddl/migrations/redshift/1458766468.sql:4]: @tspurway Yup, it's already been populated here. Redshift can't take Alembic migrations.

**Pull request closed without merging**